source/shared/cpp/ObjectModel/HostConfig.h

```diff
@@ -12,6 +12,7 @@ struct SpacingDefinition
     unsigned int right = 0;
     unsigned int top = 0;
     unsigned int bottom = 0;
+    SpacingDefinition(int left = 0, int right = 0, int top = 0, int bottom = 0):left(left), right(right), top(top), bottom(bottom){};
```

Comment:
It turns out this is necessary because the brace initialization that we use is a c++14+ feature, and the project was configured to compile with c++11. We can either add explicit constructors to every struct in the hostConfig, or change the project to use c++14 (like windows and android already do). Any thoughts on which is the better approach?